src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/src/Client/WattTimeClient.cs

```diff
@@ -17,39 +16,40 @@ namespace CarbonAware.DataSources.WattTime.Client;
 
 public class WattTimeClient : IWattTimeClient
 {
-    private static readonly JsonSerializerOptions options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
+    private static readonly JsonSerializerOptions _options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
 
-    private static readonly HttpStatusCode[] RetriableStatusCodes = new HttpStatusCode[]
+    private static readonly HttpStatusCode[] _retriableStatusCodes = new HttpStatusCode[]
     {
         HttpStatusCode.Unauthorized,
         HttpStatusCode.Forbidden
     };
 
-    private HttpClient client;
+    private HttpClient _client;
 
-    private IOptionsMonitor<WattTimeClientConfiguration> ConfigurationMonitor { get; }
+    private IOptionsMonitor<WattTimeClientConfiguration> _configurationMonitor { get; }
 
-    private WattTimeClientConfiguration Configuration => this.ConfigurationMonitor.CurrentValue;
+    private WattTimeClientConfiguration _configuration => this._configurationMonitor.CurrentValue;
 
-    private ILogger<WattTimeClient> Log { get; }
+    private ILogger<WattTimeClient> _log { get; }
 
-    private IMemoryCache memoryCache { get; }
+    private IMemoryCache _memoryCache { get; }
 
     public WattTimeClient(IHttpClientFactory factory, IOptionsMonitor<WattTimeClientConfiguration> configurationMonitor, ILogger<WattTimeClient> log, IMemoryCache memoryCache)
     {
-        this.client = factory.CreateClient(IWattTimeClient.NamedClient);
-        this.ConfigurationMonitor = configurationMonitor;
-        this.Log = log;
-        this.client.BaseAddress = new Uri(this.Configuration.BaseUrl);
-        this.client.DefaultRequestHeaders.Accept.Clear();
-        this.client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json));
-        this.memoryCache = memoryCache;
+        _client = factory.CreateClient(IWattTimeClient.NamedClient);
+        _configurationMonitor = configurationMonitor;
+        _log = log;
+        _configuration.Validate();
```

Comment:
🚗 -by: noticed we accidentally removed the `.Validate()` check from WattTimeClient